<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>ColorSamplerOptionsWidget</class>
 <widget class="QWidget" name="ColorSamplerOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Sampler</string>
  </property>
  <layout class="QGridLayout">
   <item column="0" row="6">
    <layout class="QHBoxLayout" name="blendLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="blendLabel">
       <property name="text">
        <string>Blend: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisSliderSpinBox" name="blend" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;nobr&gt;Blending controls the percentage of color that is sampled&lt;/nobr&gt; and mixed with your current brush color. A full blending value samples colors completely, ignoring brush color.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="0" row="0">
    <widget class="QComboBox" name="cmbSources">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>32767</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;nobr&gt;Determines whether the color sampler will sample&lt;/nobr&gt; colors from all visible layers or only the currently selected layer.</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Sample All Visible Layers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sample Current Layer</string>
      </property>
     </item>
    </widget>
   </item>
   <item column="0" row="8">
    <widget class="QCheckBox" name="cbNormaliseValues">
     <property name="toolTip">
      <string>Displays per-channel color information below as percentages instead of bytes.</string>
     </property>
     <property name="text">
      <string>Show colors as percentages</string>
     </property>
    </widget>
   </item>
   <item column="0" row="9">
    <widget class="QTreeWidget" name="listViewChannels">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Channel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="The value number each channel of sampled color listed in a table">Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item column="0" row="7">
    <widget class="Line" name="line2">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item column="0" row="5">
    <layout class="QHBoxLayout" name="radiusLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="radiusLabel">
       <property name="text">
        <string>Radius: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisIntParseSpinBox" name="radius">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;nobr&gt;Radius controls the color sampler's sampling area.&lt;/nobr&gt; Pixel colors within this radius are mixed together.</string>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>900</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="0" row="3">
    <layout class="QHBoxLayout" name="paletteLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbPalette">
       <property name="toolTip">
        <string>&lt;nobr&gt;Checking this box will add a new color swatch&lt;/nobr&gt; to a palette each time you sample a color.</string>
       </property>
       <property name="text">
        <string>Add to palette:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisSqueezedComboBox" name="cmbPalette"/>
     </item>
    </layout>
   </item>
   <item column="0" row="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbUpdateCurrentColor">
       <property name="toolTip">
        <string>&lt;nobr&gt;Controls whether the color sampler updates the&lt;/nobr&gt; current foreground or not.</string>
       </property>
       <property name="text">
        <string>Update color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="0" row="4">
    <widget class="Line" name="line1">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>KisSqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>KisSqueezedComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>